WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m

```diff
@@ -33,6 +33,7 @@
 
 NSString *const WPLegacyEditorNavigationRestorationID = @"WPLegacyEditorNavigationRestorationID";
 NSString *const WPLegacyAbstractPostRestorationKey = @"WPLegacyAbstractPostRestorationKey";
+NSString *const WPAppAnalyticsEditorSourceValueLegacy = @"legacy";
```

Comment:
Sorry for commenting at this point, the change would be to make `WPAppAnalyticsKeyEditorSource` an enum in `WPAppAnalytics.h`, so that it's an enum in Swift. Then the string values in each editor would go away.